Tests/LibWeb/Text/input/Worker/worker.mjs

```diff
@@ -1,3 +1,5 @@
+import "./worker.mjs";
```

Comment:
Wait, what's going on here? Importing `worker.mjs` in `worker.mjs`?

Reply:
The same trick as in https://github.com/LadybirdBrowser/ladybird/blob/master/Tests/LibWeb/Text/input/HTML/ModuleLoading/import-in-a-module.js
The crash occurs on any module import even if it's the same file.

Reply:
Meh it’s okay! Thanks for explaining :)